ReactWindows/ReactNative/Views/WebView/Events/WebViewLoadingStartEvent.cs

```diff
@@ -0,0 +1,48 @@
+﻿using Newtonsoft.Json.Linq;
+using ReactNative.UIManager.Events;
+using System;
+
+namespace ReactNative.Views.WebView.Events
+{
+    class WebViewLoadingStartEvent : Event
+    {
+        private readonly string _url;
+        private readonly bool _loading;
+        private readonly string _title;
+        private readonly bool _canGoBack;
+        private readonly bool _canGoForward;
+
+        public WebViewLoadingStartEvent(int viewTag, string url, bool loading, string title, bool canGoBack, bool canGoForward)
```

Comment:
See comments in WebViewLoadingFinishEvent.cs regarding consolidating this event.